### PR TITLE
Update distributed-tracing.md

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Tell us about your request. Provide a summary of the request and all versions that are affected.

1. Update the document to add the setting telemetry.feature.tracer.enabled: true which is a prerequisite to enable this feature.
2. Fix typo - typo related to the grpc configuration for traces, the class in the docs is mentioned incorrectly telemetry.otel.tracer.span.exporter.class=org.opensearch.telemetry.tracing.exporter.OtlpGrpcSpanExporterProvider, while it should have the following value telemetry.otel.tracer.span.exporter.class: io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter

What other resources are available? Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/OpenSearch/issues/13323

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7028

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _observing-your-data/trace/distributed-tracing.md [hunk 1]
```diff
@@ -124,7 +124,16 @@ testClusters {
 
 ### Enable distributed tracing
 
-Once you've enabled the feature flag, you can enable the tracer (which is disabled by default) by using the following dynamic setting that enables tracing in the running cluster:
+Once you've enabled the feature flag,
```
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:127]:
```suggestion
Once you've enabled the feature flag, do the following:
```

File: _observing-your-data/trace/distributed-tracing.md [hunk 2]
```diff
@@ -124,7 +124,16 @@ testClusters {
 
 ### Enable distributed tracing
 
-Once you've enabled the feature flag, you can enable the tracer (which is disabled by default) by using the following dynamic setting that enables tracing in the running cluster:
+Once you've enabled the feature flag, 
+
+1. Enable the tracing framework feature by using the following setting, which enables tracing in the opensearch.yaml file
```
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:129]:
```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.
```

File: _observing-your-data/trace/distributed-tracing.md [hunk 3]
```diff
@@ -124,7 +124,16 @@ testClusters {
...
+
+```bash
+telemetry.feature.tracer.enabled=true
+```
+{% include copy.html %}
+
+2. Also, enable the tracer by using the following dynamic setting that enables tracing in the running cluster:
```
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:136]:
```suggestion
2. Then, enable the tracer in the running cluster by adding the following dynamic setting:
```

**Review by hdhalter — COMMENTED** [review]
@Gaganjuneja - please review my edits and accept if they look good. Thanks!

**Review by Naarcha-AWS — APPROVED** [review]
LGTM

File: _observing-your-data/trace/distributed-tracing.md [hunk 4]
```diff
@@ -124,7 +124,16 @@ testClusters {
...
+Once you've enabled the feature flag, do the following:
+
+1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.
+
+```bash
+telemetry.feature.tracer.enabled=true
+```
+{% include copy.html %}
+
+2. Then, enable the tracer in the running cluster by adding the following dynamic setting:
```
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:136]:
```suggestion
2. Enable the tracer in the running cluster by adding the following dynamic setting:
```

File: _observing-your-data/trace/distributed-tracing.md [hunk 5]
```diff
@@ -124,7 +124,16 @@ testClusters {
 
 ### Enable distributed tracing
 
-Once you've enabled the feature flag, you can enable the tracer (which is disabled by default) by using the following dynamic setting that enables tracing in the running cluster:
+Once you've enabled the feature flag, do the following:
+
+1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file.
```
Comment (natebower) [inline _observing-your-data/trace/distributed-tracing.md:129]:
```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file:
```

**Review by natebower — COMMENTED** [review]
@hdhalter Just one minor change. Thanks!

File: _observing-your-data/trace/distributed-tracing.md (hunk 5 above)
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:129]:
```suggestion
1. Enable the tracing framework feature by adding the following setting in the `opensearch.yaml` file:
```

File: _observing-your-data/trace/distributed-tracing.md (hunk 4 above)
Comment (hdhalter) [inline _observing-your-data/trace/distributed-tracing.md:136]:
```suggestion
2. Enable the tracer in the running cluster by adding the following dynamic setting:
```

**Pull request merged**